scripts/create-secrets/askQuestions.js

```diff
@@ -326,6 +326,86 @@ const questions = [
         validate: input =>
             trim(input).length ? true : "secret cannot be empty!"
     },
+    {
+        type: "list",
+        name: "use-web-access-secret",
+        message:
+            "Do you need to setup the username & password used to access Magda web interface?",
```

Comment:
This is a bit confusing because there's two sets of username and password - maybe specifically say "Do you want to setup HTTP Basic authentication" ?